student.rb

```diff
@@ -9,6 +9,11 @@ def initialize(classroom = 'unknown')
   def play_hooky
     '¯\(ツ)/¯'
   end
+
```

Comment:
- [ ] [OPTIONAL] The classrom should be readable, kindly consider adding `attr_reader :classroom` to tour file